### ISSUE TITLE
Silent failures

### ISSUE THREAD

**Issue by trasher** [atoum/atoum#879]
It's unfortunately a quite common case: running tests fails without any error, causes can be multiple:
- a syntax error in a php file
- inheritance issue (try for example to override a protected method with a private one),
- ...

That's annoying, but syntax errors can be checked with `php -l`, and looking deep at the code permit to solve several other ones.

This time, I'm trying to run my test suite with php 8.1 on github actions and it fails silently. I have absolutely no idea what goes wrong. Tests are just running fine locally :/
See for example:
https://github.com/galette/galette/runs/4053979832?check_suite_focus=true

Any idea how to get more info on that fail? Thanks!

**Comment by cedric-anne** [comment]
This is due to `ini_set('display_errors', 0);` (see https://github.com/galette/galette/blob/develop/galette/includes/galette.inc.php#L131).

Same behaviour was experienced in GLPI (see https://github.com/glpi-project/glpi/pull/9744).

IMHO, atoum should register a proper error handler as soon as possible in the code to be able to exit with understandable error, even when `display_errors` and/or `error_reporting` settings says that error should not be shown.

**Comment by trasher** [comment]
Indeed, that is much more explcit; thank you very much @cedric-anne; I forget about that explicit `ini_set`!
